release_notes/addtl-release-notes.adoc

```diff
@@ -0,0 +1,96 @@
+
+|F::
+xref:../security/file_integrity_operator/file-integrity-operator-release-notes.adoc#file-integrity-operator-release-notes-v0[File Integrity Operator]
+
+|K::
+xref:../nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc#nodes-descheduler-release-notes[{descheduler-operator}]
+
+|L::
+xref:../observability/logging/logging_release_notes/logging-5-9-release-notes.adoc#logging-5-9-release-notes[{logging-uc}]
+
+|M::
+xref:../migration_toolkit_for_containers/release_notes/mtc-release-notes.adoc#mtc-release-notes[{mtc-full} ({mtc-short})]
+
+|N::
+xref:../observability/network_observability/network-observability-operator-release-notes.adoc#network-observability-operator-release-notes[Network Observability Operator] +
+{nbsp} +
+xref:../security/nbde_tang_server_operator/nbde-tang-server-operator-release-notes.adoc#nbde-tang-server-operator-release-notes[Network-bound Disk Encryption (NBDE) Tang Server Operator]
+
+|O::
+xref:../backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc#oadp-1-4-release-notes[{oadp-first}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_dev_spaces/3.16#Release%20Notes[{openshift-dev-spaces-productname}] +
+{nbsp} +
+xref:../observability/distr_tracing/distr-tracing-rn.adoc#distr-tracing-rn[{DTProductName}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/1.13/html/release_notes/index[{gitops-title}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_local/[{openshift-local-productname}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/openshift_sandboxed_containers/1.6/html/release_notes/index[{osc}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_serverless/1.33/html/about_openshift_serverless/serverless-release-notes#serverless-release-notes[Red Hat {ServerlessProductName}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_pipelines/1.15/html/about_openshift_pipelines/op-release-notes[{pipelines-title}] +
+{nbsp} +
+xref:../virt/release_notes/virt-release-notes-placeholder.adoc#virt-release-notes-placeholder[Red Hat {VirtProductName}] +
+{nbsp} +
+xref:../observability/otel/otel-rn.adoc#otel-rn[{OTELName}]
+
+|P::
+xref:../observability/power_monitoring/power-monitoring-release-notes.adoc#power-monitoring-release-notes[{PM-title-c}]
+
+|R::
+xref:../nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc#run-once-duration-override-release-notes[{run-once-operator}]
+
+|S::
+xref:../nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc#nodes-secondary-scheduler-release-notes[{secondary-scheduler-operator-full}] +
+{nbsp} +
+xref:../security/security_profiles_operator/spo-release-notes.adoc#spo-release-notes[Security Profiles Operator] +
+{nbsp} +
+xref:../service_mesh/v2x/servicemesh-release-notes.adoc#service-mesh-release-notes[{SMProductName} 2.x] +
```

Comment:
Shouldn't the service meshes be under "O" with the rest of the "OpenShift" ones?

Reply:
Aye, didn't notice the product name changed when I switch it to the attribute!